src/openforms/registrations/contrib/objects_api/config.py

```diff
@@ -49,7 +53,16 @@ class ObjecttypeVariableMappingSerializer(serializers.Serializer):
 
 
 class ObjectsAPIOptionsSerializer(JsonSchemaSerializerMixin, serializers.Serializer):
-
+    objects_api_group = PrimaryKeyRelatedAsChoicesField(
+        queryset=ObjectsAPIGroupConfig.objects.exclude(
+            objects_service=None,
+            objecttypes_service=None,
+            drc_service=None,
+            catalogi_service=None,
+        ),
```

Comment:
this only excludes the groups where *all* of the fields are `NULL`, not where *any* of the fields are `NULL`. For that, you'd need to either chain the `.exclude(field=None)` calls or use a `Q(objects_service__isnull=True) | Q(objecttypes_service__isnull=True) | ...` in the `.filter` call.